### PR TITLE
Update to latest css-layout changes

### PR DESCRIPTION
**Pull request by mcudich**

### CODE REVIEW

**Comment by mcudich** [comment]
Friendly ping on this :)

**Comment by joshaber** [comment]
Thanks! I've been sick so this was put on the back burner, but it's on my list :smile:

**Comment by mcudich** [comment]
No problem! Looking forward to your feedback.

File: SwiftBox/Node.swift [hunk 1]
```diff
@@ -30,12 +38,16 @@ public struct Edges {
 		self.right = uniform
 		self.bottom = uniform
 		self.top = uniform
+		self.start = uniform
+		self.end = uniform
 	}
 }
 
-public enum Direction: UInt32 {
+public enum FlexDirection: UInt32 {
```
Comment (joshaber) [inline SwiftBox/Node.swift]: Why rename this one?

Comment (joshaber) [inline SwiftBox/Node.swift]: Oh never mind, I see!

**Comment by joshaber** [comment]
Thanks for doing the legwork on this! :metal:

**Pull request merged**